### ISSUE TITLE
aws_ecs_service placement_strategy documentation update

### ISSUE THREAD

**Issue by mdkent** [hashicorp/terraform#11844]
Quick one here: the aws_ecs_service placement_strategy response from the API was lowercased in #11644, but the documentation [here](https://www.terraform.io/docs/providers/aws/r/ecs_service.html#placement_strategy) says that `instanceId` is a valid value, which doesn't work when it gets lowercased. Suggest updating the docs to `instanceid`, or stop lowercasing this value.

### Terraform Version
0.8.6.

### Affected Resource(s)
- aws_ecs_service

### Terraform Configuration Files
```hcl
resource "aws_ecs_service" "service_test" {
  name            = "test"
  cluster         = "${aws_ecs_cluster.ecs.id}"
  desired_count   = "${var.main_cluster["autoscale_min"]}"
  task_definition = "${aws_ecs_task_definition.task_test.arn}"

  deployment_minimum_healthy_percent = 50
  deployment_maximum_percent = 100

  placement_strategy {
    type  = "spread"
    field = "instanceId"
  }
}
```

### Plan Output
```
-/+ module.ecs.aws_ecs_service.service_test
    cluster:                             "arn:aws:ecs:us-east-1:178556630403:cluster/test-ecs" => "arn:aws:ecs:us-east-1:178556630403:cluster/test-ecs"
    deployment_maximum_percent:          "100" => "100"
    deployment_minimum_healthy_percent:  "50" => "50"
    desired_count:                       "10" => "10"
    name:                                "test" => "test"
    placement_strategy.#:                "1" => "1"
    placement_strategy.1676812570.field: "instanceid" => "" (forces new resource)
    placement_strategy.1676812570.type:  "spread" => "" (forces new resource)
    placement_strategy.3946258308.field: "" => "instanceId" (forces new resource)
    placement_strategy.3946258308.type:  "" => "spread" (forces new resource)
    task_definition:                     "arn:aws:ecs:us-east-1:178556630403:task-definition/test:8" => "arn:aws:ecs:us-east-1:178556630403:task-definition/test:8"
```

### Expected Behavior
Nothing, idempotent plan.

### Actual Behavior
terraform wants to recreate the service on each run.

### Steps to Reproduce
1. `terraform push`

### References
- #11644

**Comment by mdkent** [comment]
Ugh, this goes a bit deeper, using `instanceid` on a newly created service results in

```
* aws_ecs_service.service_large_logspout: InvalidParameterException: spread field 'instanceid' is invalid.
	status code: 400, request id: ea4c9b36-ef5b-11e6-9612-859c0caa2152
```

**Comment by mdkent** [comment]
Deeper still: `host` as suggested in the [documentation](https://www.terraform.io/docs/providers/aws/r/ecs_service.html#placement_strategy) doesn't work either. The API returns `instanceId` again, which gets converted to `instanceid`.

```
    placement_strategy.#:                      "1" => "1"
    placement_strategy.1427850681.field:       "" => "host" (forces new resource)
    placement_strategy.1427850681.type:        "" => "spread" (forces new resource)
    placement_strategy.1676812570.field:       "instanceid" => "" (forces new resource)
    placement_strategy.1676812570.type:        "spread" => "" (forces new resource)
```

**Comment by Z3r0Sum** [comment]
I'm also running into this issue.

**Comment by KensoDev** [comment]
@Z3r0Sum check #12829

**Comment by stack72** [comment]
Closed via #12998
